### PR TITLE
certik 8.9.22

### PR DESCRIPTION
**Pull request by riley-stride**
See skyharbor for description, everything is pretty nit

### CODE REVIEW

File: x/records/keeper/deposit_record.go [hunk 1]
```diff
@@ -75,6 +76,12 @@ func (k Keeper) GetDepositRecord(ctx sdk.Context, id uint64) (val types.DepositR
 func (k Keeper) RemoveDepositRecord(ctx sdk.Context, id uint64) {
 	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.DepositRecordKey))
 	store.Delete(GetDepositRecordIDBytes(id))
+	count := k.GetDepositRecordCount(ctx)
+	if count > 0 {
```
Comment (asalzmann) [inline x/records/keeper/deposit_record.go:80]: flagging as a bug (per offline discussion) that we should not merge in

**Review by asalzmann — CHANGES_REQUESTED** [review]

File: x/records/keeper/deposit_record.go (hunk 1 above)
Comment (riley-stride) [reply x/records/keeper/deposit_record.go]: reverted, ty

File: x/interchainquery/keeper/abci.go [hunk 2]
```diff
@@ -11,10 +11,6 @@ import (
 	"github.com/Stride-Labs/stride/x/interchainquery/types"
 )
 
-const (
-	RetryInterval = 25
```
Comment (asalzmann) [inline x/interchainquery/keeper/abci.go:15]: Is this just an unused variable? If so, good to merge

**Review by asalzmann — APPROVED** [review]

File: x/interchainquery/keeper/abci.go (hunk 2 above)
Comment (riley-stride) [reply x/interchainquery/keeper/abci.go:15]: yes

**Pull request merged**